services/ansible/ansible.tester.js

```diff
@@ -26,3 +28,13 @@ t.create('role downloads (valid)')
 t.create('role downloads (not found)')
   .get('/role/d/does-not-exist.json')
   .expectJSON({ name: 'role downloads', value: 'not found' })
+
+t.create('quality score (valid)')
```

Comment:
Since you've split the original service file into `roles` and `quality` files, can you do the same for the tester files (at least for `quality` via a `ansible-quality.tester.js`?

Reply:
Thanks for your feedbacks @calebcartwright. I started to split the tester files, but there are two services in the `ansible-roles.service.js` (`AnsibleGalaxyRoleDownloads` and `AnsibleGalaxyRoleName`). Should I also split this service file in two files? And of course split testers.

I ask this question because I tried to use the 
`const t = (module.exports = require('../create-service-tester')())` declaration and compiler throw me an error.

Reply:
Happy to help!

> compiler throw me an error

I'm not sure what you mean there. Are you getting an error when you try to launch the tests, or are the tests running but failing with an error message?

Reply:
The `create-service-tester` shortcut assumes a file exports only one service. If your file exports more than one service then you need to create a new `ServiceTester` object for it.

One way to do it is have `ansible-roles.service.js` export 2 services and we create the tester object using  `const t = (module.exports = new ServiceTester({ id: 'ansible', title: 'Ansible Galaxy Role' }))`

The other way would be to move the shared code to a helper file and split into 2 service files each with a corresponding tester then make the tester object using `const t = (module.exports = require('../create-service-tester')())` and naming convention.

I don't have a strong preference either way really.